### DISCUSSION TITLE
Where are the planets?

### DISCUSSION THREAD

**Discussion by NABS2ER** [terrarium-earth/Ad-Astra#504] [Q&A]
After update 1.15.5, planets such as Jupiter, Saturn, Uranus, and Neptune disappeared. Were they removed on purpose or is there some other way to get to them now?
![2024-03-04_13 04 23](https://github.com/terrarium-earth/Ad-Astra/assets/141981213/d3bbfc3f-53b1-4f4f-8de2-1b29d282c09e)

This is what update 1.15.18 looks like
![2024-03-04_14 05 42](https://github.com/terrarium-earth/Ad-Astra/assets/141981213/e5ed6d87-47b6-490b-9561-83e184606d21)

**Comment by AlexNijjar** [comment]
those planets don't exist. probably some addon. and 1.15.5 isn't the latest

**Reply by NABS2ER** [reply]
This is really an add-on, but then it turns out that add-ons do not work on new versions. Since I only updated ad astra and botarium to the latest version. And the planets disappeared!

**Reply by AlexNijjar** [reply]
This is not our issue. report it to whoever made the addon
